tests/e2e_tests/docker-compose.e2e-tests.yml

```diff
@@ -0,0 +1,16 @@
+version: '2'
+
+services:
+  mender-gui-tests-runner:
+    build:
+      dockerfile: tests/e2e_tests/Dockerfile.gui
+      context: .
+    networks:
+      - mender
+    depends_on:
+      - mender-api-gateway
+      - mender-gui
+  mender-gui:
+    build:
+      dockerfile: ${GUI_REPOSITORY}/Dockerfile
+      context: ${GUI_REPOSITORY}
```

Comment:
I definitely need to learn `docker-compose` :wink:

This setup covers a very similar need I have with the `.deb` packaging task and I gave up (using `docker run --network ...` directly from my tests).

Reply:
let me know whenever you have a question :)